frontend/src/pages/projects/screens/projects/ProjectsToolbar.tsx

```diff
@@ -14,57 +30,165 @@ type ProjectsToolbarProps = {
   allowCreate: boolean;
   filterData: ProjectsFilterDataType;
   onFilterUpdate: (key: string, value?: string | { label: string; value: string }) => void;
+  aiProjectNum: number;
+  fullProjectNum: number;
+  setProjectFilter: (projectFilter: string) => void;
 };
 
+export const AiLabel: React.FC = () => (
+  <Label
+    icon={<OutlinedStarIcon />}
+    variant="outline"
+    data-testid="ai-project-label"
+    style={{ marginLeft: '5px' }}
```

Comment:
Please add `isCompact` to the label (This is shown in the mockup). Also, please avoid using the custom styles like this according to the [best practice](https://github.com/opendatahub-io/odh-dashboard/blob/main/docs/best-practices.md#avoid-custom-styling) doc. Because this label could be reused in the future, or if PF changes the components, it will be hard to debug the style issues.
Instead, use `Flex` layout whenever you need to create a gap between 2 components. In this case, whenever you are using the `AiLabel` component, you should wrap a <Flex> around the title and the component.

Reply:
it's just a margin tho.  adding flex layouts make it much more complicated.  the main concern with styles is dark mode; not margins